### PR TITLE
Added an example to the start_link documentation

### PR DESCRIPTION
**Pull request by k-cross**
I didn't RTFM close enough and it was listed, but I think having an example would still be beneficial for non-default configuration options.

Code is all functionally the same. Close it if you feel like it's noise. Cheers!

### CODE REVIEW

File: lib/finch.ex [hunk 1]
```diff
@@ -80,6 +84,22 @@ defmodule Finch do
...
+    {
+      Finch,
+      name: MyFinch,
+      pools: %{
+        default: [
+          size: 10,
+          count: 1,
+          protocol: :http1,
+          max_idle_time: :infinity,
+          conn_opts: [transport_opts: [ciphers: :ssl.cipher_suits(:all)]]
```
Comment (wojtekmach) [inline lib/finch.ex:99]: please don’t use :ssl.cipher_suits/1 as it’s going away in OTP 24 :)

Comment (k-cross) [reply lib/finch.ex]: Updated to used `:ssl.cipuer_suits/2`

**Comment by sneako** [comment]
I generated the docs from this PR and the example that is added felt a bit repetitive to me, since we already have a similar example in the usage section, see [here](https://hexdocs.pm/finch/Finch.html). Also, the spacing is a bit off so the code formatting is slightly broken, see screenshot.
Screen Shot 2021-02-16 at 09 23 28

 Do you thinking adding a few more configuration options to the existing example in the "Usage" section could be as effective?

**Comment by k-cross** [comment]
@sneako yeah, that works too. I must have originally missed the example in the main page. It'd be great if there was a place to put the same example in both places but in a way that avoids duplication but that's probably overkill. I do think it's nice to have examples in both the functions that describe the behavior as well as top level since it is about general setup.

I'll make the change once I can get back to my personal computer.

**Comment by sneako** [comment]
I'm going to close this due to inactivity, but please feel free to submit a new PR!

**Pull request closed without merging**